### ISSUE TITLE
Logger issue

### ISSUE THREAD

**Issue by RusKryzhanovskiy** [aissat/easy_localization#241]
Could you add an option to disable standard logging?
```
[Easy Localization] Start
[Easy Localization] Init state
[Easy Localization] Build
[Easy Localization] Saved locale loaded en_GB
[Easy Localization] Load asset from assets/locales
[Easy Localization] Init Localization Delegate
[Easy Localization] Init provider
``` 
Add either `bool enableLogging` or `Function(String message) onLog` so that I can decide how to do the logging.

**Comment by alexaf2000** [comment]
An error handler will be also a good idea. I use Firebase Crashlytics to notify an issue in my application remotely when an error is throw. But as there is no OnError method or something like this, can't be used to detect EasyLocalization errors.

Would be a great feature.

**Comment by Overman775** [comment]
@RusKryzhanovskiy  @alexaf2000  PR #315

**Comment by Overman775** [comment]
@RusKryzhanovskiy @alexaf2000 released #315
